extensions/brew/src/utils.ts

```diff
@@ -1,15 +1,20 @@
 import { Clipboard, environment, Toast } from "@raycast/api";
 import path from "path";
-import { mkdirSync } from "fs";
-import { stat, readFile, writeFile } from "fs/promises";
+import fs from "fs";
+import { stat } from "fs/promises";
 import fetch, { FetchError } from "node-fetch";
+import { chain } from "stream-chain";
```

Comment:
I'm assuming `stream-chain` is a dependency of `stream-json`? In any case, I would add it as a direct dependency

Reply:
Yes, that's correct. Will add it 👍